src/templates/en/2021/methodology.html

```diff
@@ -0,0 +1,474 @@
+
+    <section>
+      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>
+
+      <p>
+        In May and June {{ year }}, with the stable list of metrics and chapters, data analysts triaged the metrics for feasibility. In some cases, <a hreflang="en" href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">custom metrics</a> were created to fill gaps in our analytic capabilities.
+      </p>
+
+      <p>
+        Throughout July {{ year }}, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
+      </p>
+
+      <p>
+        The data analysts began writing queries to extract the results for each metric. In total, hundreds of queries were written by hand! You can browse all of the queries by year and chapter in our open source <a hreflang="en" href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/{{ year }}">query repository</a> on GitHub.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="interpretation"><a href="#interpretation" class="anchor-link">Interpretation</a></h3>
+
+      <p>
+        Authors worked with analysts to correctly interpret the results and draw appropriate conclusions. As authors wrote their respective chapters, they drew from these statistics to support their framing of the state of the web. Peer reviewers worked with authors to ensure the technical correctness of their analysis.
+      </p>
+
+      <p>
+        To make the results more easily understandable to readers, web developers and analysts created data visualizations to embed in the chapter. Some visualizations are simplified to make the points more clearly. For example, rather than showing a full distribution, only a handful of percentiles are shown. Unless otherwise noted, all distributions are summarized using percentiles, especially medians (the 50th percentile), and not averages.
+      </p>
+
+      <p>
+        Finally, editors revised the chapters to fix simple grammatical errors and ensure consistency across the reading experience.
+      </p>
+    </section>
+
+    <section>
+      <h2 id="looking-ahead"><a href="#looking-ahead" class="anchor-link">Looking ahead</a></h2>
+
+      <p>
+        The {{ year }} edition of the Web Almanac is the third in what we hope to continue as an annual tradition in the web community of introspection and a commitment to positive change. Getting to this point has been a monumental effort thanks to many dedicated <a href="{{ url_for('contributors', year=year, lang=lang) }}">contributors</a> and we hope to leverage as much of this work as possible to make future editions even more streamlined.
+      </p>
+
+      <p>
+        If you&#8217;re interested in contributing to the 2022 edition of the Web Almanac, please fill out our <a hreflang="en" href="https://forms.gle/55uatdX9T3JZG2837">interest form</a>. Let&#8217;s work together to track the state of the web!
```

Comment:
Updated the link based on this tweet: https://twitter.com/HTTPArchive/status/1448727421813936131

Can you double check it's accurate @rviscomi ?

Reply:
LGTM